src/universal/components/ProjectEditor/ProjectEditor.js

```diff
@@ -11,6 +11,7 @@ import withKeyboardShortcuts from './withKeyboardShortcuts';
 import withLinks from './withLinks';
 import withSuggestions from './withSuggestions';
 import entitizeText from 'universal/utils/draftjs/entitizeText';
+import './Draft.css';
```

Comment:
This css file fails a production build. See: https://circleci.com/gh/ParabolInc/action/2904